build.gradle.kts

```diff
@@ -42,14 +58,21 @@ spotless {
     // See the License for the specific language governing permissions and
     // limitations under the License.
     //===----------------------------------------------------------------------===//
-  """.trimIndent()
+  """
+      .trimIndent()
   java {
     target("**/*.java")
     licenseHeader(starBlockLicenseHeader)
   }
   kotlin {
-    target("**/*.kt")
     licenseHeader(starBlockLicenseHeader)
+    target("src/**/*.kt", "buildSrc/**/*.kt")
```

Comment:
Why was this changed?

Reply:
I modeled most of the gradle changes off pkl-pantry and that's what's there.